drydock/manifest_builder/infrastructure/tutor_config.py

```diff
@@ -1,14 +1,61 @@
 from abc import abstractmethod
-from tutor import config as tutor_config
+import tutor
+from tutor.exceptions import TutorError
 from drydock.manifest_builder.domain.config import DrydockConfig
 
 
 class TutorConfig(DrydockConfig):
-
     def __init__(self, context, options: dict):
         self.context = context
         self.options = options
+        self.defaults = {
+            "DRYDOCK_FLOWER": False,
+            "DRYDOCK_INGRESS": False,
+            "DRYDOCK_HPA": False,
+            "DRYDOCK_DISABLE_JOBS": False,
+            "DRYDOCK_ENABLE_OVERRIDES": False,
+            "DRYDOCK_LMS_MIN_REPLICAS": 1,
+            "DRYDOCK_CMS_MIN_REPLICAS": 3,
+            "DRYDOCK_LMS_MAX_REPLICAS": 1,
+            "DRYDOCK_CMS_MAX_REPLICAS": 3,
+            "DRYDOCK_LMS_LIMIT_CPU": 1,
+            "DRYDOCK_LMS_LIMIT_MEMORY": "2Gi",
+            "DRYDOCK_LMS_REQUEST_CPU": "600m",
+            "DRYDOCK_LMS_REQUEST_MEMORY": "1Gi",
+            "DRYDOCK_LMS_WORKER_LIMIT_CPU": 1,
+            "DRYDOCK_LMS_WORKER_LIMIT_MEMORY": "2Gi",
+            "DRYDOCK_LMS_WORKER_REQUEST_CPU": "600m",
+            "DRYDOCK_LMS_WORKER_REQUEST_MEMORY": "1Gi",
+            "DRYDOCK_CMS_LIMIT_CPU": 1,
+            "DRYDOCK_CMS_LIMIT_MEMORY": "2Gi",
+            "DRYDOCK_CMS_REQUEST_CPU": "600m",
+            "DRYDOCK_CMS_REQUEST_MEMORY": "1Gi",
+            "DRYDOCK_CMS_WORKER_LIMIT_CPU": 1,
+            "DRYDOCK_CMS_WORKER_LIMIT_MEMORY": "2Gi",
+            "DRYDOCK_CMS_WORKER_REQUEST_CPU": "600m",
+            "DRYDOCK_CMS_WORKER_REQUEST_MEMORY": "1Gi",
+        }
```

Comment:
self.defaults changes? why this dictionary is not constant to the class?

Reply:
I've opted to define the defaults on a `defaults.yml` on each template set and load them from there.